ui/app/templates/vault/cluster/access/oidc/providers/provider/details.hbs

```diff
@@ -0,0 +1,49 @@
+<Toolbar>
+  <ToolbarActions>
+    {{#if this.model.canDelete}}
+      <ToolTip @verticalPosition="above" as |T|>
+        <T.Trigger tabindex="-1">
+          <ConfirmAction
+            @disabled={{eq this.model.name "default"}}
+            @buttonClasses="toolbar-link"
+            @onConfirmAction={{this.delete}}
+            @confirmTitle="Delete provider?"
+            @confirmMessage="This provider will be permanently deleted. You will need to re-create it to use it again."
+            @confirmButtonText="Delete"
+          >
+            Delete provider
+          </ConfirmAction>
+        </T.Trigger>
+        {{#if (eq this.model.name "default")}}
+          <T.Content @defaultClass="tool-tip smaller-font">
+            <div class="box">
+              This is a built-in provider that cannot be deleted.
+            </div>
+          </T.Content>
+        {{/if}}
+      </ToolTip>
+      <div class="toolbar-separator"></div>
+    {{/if}}
+    {{#if this.model.canEdit}}
+      <ToolbarLink @params={{array "vault.cluster.access.oidc.providers.provider.edit" this.model.name}}>
+        Edit provider
+      </ToolbarLink>
+    {{/if}}
+  </ToolbarActions>
+</Toolbar>
+<div class="box is-fullwidth is-sideless is-paddingless is-marginless">
+  <InfoTableRow @label="Name" @value={{this.model.name}} @alwaysRender={{true}} />
+  <InfoTableRow @label="Issuer URL" @value={{this.model.issuer}} @addCopyButton={{true}} @alwaysRender={{true}} />
+  <InfoTableRow @label="Scopes">
+    {{#if (gt @model.scopesSupported.length 0)}}
+      {{#each @model.scopesSupported as |scope|}}
+        <LinkTo @route="vault.cluster.access.oidc.scopes.scope.details" @model={{scope}}>
```

Comment:
Any chance we'll have 10 or more scopes? If so, in transform or database if there were more than 10, like a big array that needed links to each we used the info-table-item-array. It's what it's designed for. Handling an array of items that go into a LinkTo in an InfoTableRow.

Reply:
Actually @yankun91  - what are your thoughts here? Do you think we'll have enough scopes to need a `View all` option here like we added in assignments for groups and entities?

Reply:
Great - thank you!